config/webpack.common.js

```diff
@@ -82,36 +88,59 @@ module.exports = {
 
       {
         test: /\.css$/,
-        use: [
-          {
-            loader: 'to-string-loader'
-          },
-          // Commented out as causing CSS ot load twice
-          /*{
-           loader: 'style-loader'
-           },*/
-          {
-            loader: 'css-loader'
-          },
-        ]
+        loader: extractCSS.extract({
+          fallback: "style-loader",
+          use: "css-loader?sourceMap&context=/"
+        })
       },
+
       {
         test: /\.scss$/,
-        use: [
+        loaders: [
```

Comment:
Why change this to "loaders", under webpack 2.2 it should be "use". The allow "loaders" for backwards compatibility but it is no longer the standard.